Dockerfile.soroban-cli

```
@@ -6,6 +6,7 @@ FROM $DOCKERHUB_RUST_VERSION AS builder
 WORKDIR /soroban-tools
 COPY . .
 
+run apt update && apt install -y libdbus-1-dev
```

```suggestion
run apt update && apt install -y libdbus-1-dev libudev-dev
```
Can we add the dep needed for ledger while we are at it?